sensor_node:
  ros__parameters:
    publish_frequency: 2.0

nav_node:
  ros__parameters:
    front_distance_threshold: 0.7
    linear_speed: 0.3
    angular_speed: 0.8

motor_node:
  ros__parameters:
    motor_name: "my_left_motor"
